Src/Newtonsoft.Json/Converters/BsonObjectIdConverter.cs

```diff
@@ -33,26 +33,24 @@ namespace Newtonsoft.Json.Converters
     /// <summary>
     /// Converts a <see cref="BsonObjectId"/> to and from JSON and BSON.
     /// </summary>
-    public class BsonObjectIdConverter : JsonConverter
+    public class BsonObjectIdConverter : JsonConverter<BsonObjectId>
```

Comment:
So JsonConverter<T> inherits from JsonConverter, but is changing the base type to JsonConverter<T> a breaking change?